api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -249,6 +249,8 @@ message RemoteJwks {
   // * Jwks is ready when the requests come, not need to wait for the Jwks fetching.
   //
   JwksAsyncFetch async_fetch = 3;
+
+  config.core.v3.RetryPolicy retry_policy = 4;
```

Comment:
Please add comment on when it applies, e.g.  it will apply both async_fetch and non async_fetch.

Reply:
done

Reply:
not quite sure why the docs do not get generated though. [See this pipeline results](https://github.com/envoyproxy/envoy/pull/16924/checks?check_run_id=2851924986).